pages/desktop/regions/filter.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, testsetup, lookup):
 
             # expand the thing here to represent the proper user action
             self._root_element = self.selenium.find_element(self._base_locator[0],
-                                    "%s/ul/li/a[normalize-space(text())='%s']" % (self._base_locator[1], lookup))
+                                                            "%s/ul/li/a[normalize-space(text())='%s']" % (self._base_locator[1], lookup))
```

Comment:
+1 on the breaking line
-1 on the string formating as we want consistency in the project.

If we want to move to the new fring formating you can make a task and update the full project :smile_cat: